Example/Tests/GINICameraViewControllerTests.swift

```diff
@@ -3,7 +3,7 @@ import XCTest
 
 class CameraViewControllerTests: XCTestCase {
     
-    var vc = CameraViewController(success: { _ in }, failure: { _ in })
+    var vc = CameraViewController(successBlock: { _ in }, failureBlock: { _ in })
```

Comment:
Why did you choose `*Block` instead of just success or failure? Just curious...

Reply:
Because otherwise there will be an ambiguous reference and Xcode does not know how to solve it.

Reply:
Wont compile, or code completion wouldn't work?

Reply:
It won't compile. Since both parameters are `closures` in both methods, Xcode does not know what is the difference.